optionals/compat_sog/CfgVehicles/land.hpp

```diff
@@ -0,0 +1,28 @@
+
+// fuel pumps
+class Land_vn_building_base;
+class Land_vn_commercial_base : Land_vn_building_base {};
+
+class Land_vn_fuelstation_01_pump_f : Land_vn_commercial_base {
```

Comment:
`Land_vn_building_base` import is useless.
```suggestion
class Land_vn_commercial_base;
class Land_vn_fuelstation_01_pump_f : Land_vn_commercial_base {
```